<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-07T21:49:44 ICT" name="Default suite" finished-at="2022-09-07T21:51:14 ICT" duration-ms="90069">
    <groups>
    </groups>
    <test started-at="2022-09-07T21:49:44 ICT" name="Default test" finished-at="2022-09-07T21:51:14 ICT" duration-ms="90069">
      <class name="tests.WebTableTest">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:tests.WebTableTest@1fe20588]" started-at="2022-09-07T21:49:44 ICT" name="openBrowser" finished-at="2022-09-07T21:49:56 ICT" duration-ms="12484" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="verifyTextAfterSubmit(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:tests.WebTableTest@1fe20588]" started-at="2022-09-07T21:49:56 ICT" name="verifyTextAfterSubmit" data-provider="RegistrationData" finished-at="2022-09-07T21:51:14 ICT" duration-ms="77570" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Lonh]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Hoa]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[linh@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[20]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[2000]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[bonee]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Lonh] but found [Cierra]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Lonh] but found [Cierra]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at tests.WebTableTest.verifyTextAfterSubmit(WebTableTest.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTextAfterSubmit -->
      </class> <!-- tests.WebTableTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
